code_of_conduct.md

```diff
@@ -0,0 +1,180 @@
+
+In your report, please do your best to include:
+
+- Your contact information
+- Identifying information of the reported person
+- The behaviour associated with the Code of Conduct violation
+- If possible, where the Code of Conduct violation happened
+- The approximate time of the behaviour (if different than the time the report was made)
+- The circumstances surrounding the incident
+- Other people involved or that witnessed the incident
+- If there is a publicly available record (i.e. email thread, Slack messages, Twitter threads)
+
+If you believe this is an ongoing situation, please let us know
+- Any additional helpful information
+
+## What to do if someone is in immediate danger?
+
+If you believe someone is in immediate danger, please ask a Council member to contact appropriate emergency responders. All event organizers should, before the event, determine who would be appropriate to contact in case of an incident and make sure this information is available at all times over the duration of the event in question.
+
+Once the incident has been resolved, we ask that it be reported to the Council in the same way as all other incidents.
+
+## Enforcement manual
+The following section details the enforcement manual followed by the Council. It is used when we respond to an issue to make sure we are consistent and fair on every instance. Enforcement of the Code of Conduct should be respectful and not include any harassing behaviours.
+
+All responses to reports of conduct violations will be managed by the Council itself. To write to the Council and for the current membership, see [https://rse-aunz.github.io/about/](https://rse-aunz.github.io/about/)
+
+### How will the Council respond to reports?
+When a report is sent to the Council, a member will reply to the report to confirm receipt within 24 hours. If a report does not contain enough information, the Council will attempt to obtain all relevant data before acting, including contacting any individuals involved to get a comprehensive account of events. Once a report has been filed, one of the Council members will be assigned as the main acting member responsible for any communications with the involved individual(s).
+
+The Council is also empowered to act if any of its members becomes aware of ongoing behaviour that, taken as a whole over a long time period, is disrupting or harassing. Such behaviour might not be “over the line” in any single incident, and thus may not generate a report.
+
+### Immediate response
+The initial response to an incident is very important. Depending on the severity and details of the incident, an immediate response may be required.
+
+In situations involving immediate danger or involving a threat to anyone’s safety, any member of the Council may – and should – act immediately to protect the safety of the individual(s) in such a situation. This can include contacting law enforcement or crisis resources.
```

Comment:
```suggestion
In situations involving immediate danger or involving a threat to anyone’s safety, any member of the RSE-AUNZ may – and should – act immediately to protect the safety of the individual(s) in such a situation. This can include contacting law enforcement or crisis resources.
```